src/ServicePulse.Host/vue/src/components/EventLogItem.vue

```diff
@@ -0,0 +1,186 @@
+<script setup>
+import { useRouter } from "vue-router";
+import TimeSince from "../components/TimeSince.vue";
+
+defineProps({
+  eventLogItem: Object,
+});
+const router = useRouter();
+
+function navigateToEvent(eventLogItem) {
+  switch (eventLogItem.category) {
+    case "Endpoints":
```

Comment:
suggestion to create eventLogItemCategory as a frozen object containing all the possible values, so that spelling mistakes or changes become a non-issue (see flowdiagram for an example)

Reply:
same for other string comparisons below